### PR TITLE
feat:modal component

### PR DESCRIPTION
**Pull request by ultronjs**
I have created a modal preview class-exclusive for the library as modal by default should have a fixed position and I don't want to create confusion for the user. Thatswhy you see same code in CSS for modal_preview and modal class.

### CODE REVIEW

File: css/pages/pagesStyle.css [hunk 1]
```diff
@@ -67,6 +67,7 @@ body {
     position: fixed;
     top: 70px;
```
Comment (ozairkhan2810) [inline css/pages/pagesStyle.css]: use consistent units

File: css/pages/pagesStyle.css [hunk 2]
```diff
@@ -142,12 +143,14 @@ iframe{
 /*footer*/
 .footer {
     grid-area: footer;
-    background-color: var(--gre);
+    background-color: var(--white);
     padding: 0.75rem 1rem;
     text-align: center;
     color: var(--black);
     border-top-right-radius: 1rem;
     border-top: 1px solid black;
```
Comment (ozairkhan2810) [inline css/pages/pagesStyle.css]: use black as color-variable

**Review by ozairkhan2810 — APPROVED** [review]
looks good, just maintain blank space between css blocks
can merge

File: css/pages/pagesStyle.css (hunk 1 above)
Comment (ultronjs) [reply css/pages/pagesStyle.css]: done

File: css/pages/pagesStyle.css (hunk 2 above)
Comment (ultronjs) [reply css/pages/pagesStyle.css]: done

**Review by ajit-sharmaGH — APPROVED** [review]
Everything looks fine to me, code is clean !

**Comment by ultronjs** [comment]
Merging the PR after 2 approvals

**Pull request merged**